README.md

```diff
@@ -62,12 +62,13 @@ sudo apt-get install make cmake g++ libboost-dev libboost-system-dev \
   libbz2-dev libpq-dev libproj-dev lua5.2 liblua5.2-dev
 ```
 
-On a Fedora system, use
+On a Fedora 24/25 system, use
 
 ```sh
-sudo yum install cmake gcc-c++ boost-devel expat-devel zlib-devel bzip2-devel \
-  postgresql-devel proj-devel proj-epsg lua-devel
+sudo dnf install cmake make gcc-c++ boost-devel expat-devel zlib-devel bzip2-devel \
+  postgresql-devel proj-devel proj-epsg lua-devel geos-devel
 ```
+_Note: Fedora 26 uses GEOS 3.6.1 which is currently incompatible.  See [Issue 634](https://github.com/openstreetmap/osm2pgsql/issues/634)_
```

Comment:
GEOS is no longer a requirement, so this comment is not needed. Neither is the added geos-devel dependency.

Reply:
I built today and can report that the current build instructions are not sufficient to get a working build on Fedora 24/25.   `cmake ..` will fail saying that GEOS is required.  Does this mean the make files need to be updated to remove the GEOS requirement?

I tested on Docker containers `fedora:24` and `fedora:25`.

Reply:
Did you try build master or a release?

Reply:
This was on `0.92.0` branch

Reply:
Master works as expected, I will remove the GEOS note and devel library